tests/monotouch-test/CoreImage/FilterTest.cs

```diff
@@ -99,6 +103,10 @@ public void ColorSpace ()
 			using (var f = new CIColorCubeWithColorSpace ()) {
 				Assert.Null (f.ColorSpace, "ColorSpace/default");
 				using (var cs = CGColorSpace.CreateDeviceGray ()) {
+					var nsname = NSString.CreateNative ("inputColorSpace");
+					// TODO: remove when `CIBarcodeDescriptorTest` passes
+					// This works `inputColorSpace` is set
+					Messaging.void_objc_msgSend_IntPtr_IntPtr (f.Handle, Selector.GetHandle ("setValue:forKey:"), cs.Handle, nsname);
```

Comment:
This leaks `nsname`, you need to `NSString.ReleaseNative (nsname)`.

Reply:
Yes yes but this code is going to be removed: `TODO: remove when 'CIBarcodeDescriptorTest' passes`.

It's just to illustrate that I can't get `setValue:forKey:` to work for `CIBarcodeGenerator` when it works fine for `CIColorCubeWithColorSpace`.

Reply:
Gotcha, clearly I just read what I want to read and ignore the rest..